lib/index.js

```diff
@@ -258,7 +258,35 @@ module.exports = {
     'strongly-recommended': require('./configs/strongly-recommended'),
     'vue3-essential': require('./configs/vue3-essential'),
     'vue3-recommended': require('./configs/vue3-recommended'),
-    'vue3-strongly-recommended': require('./configs/vue3-strongly-recommended')
+    'vue3-strongly-recommended': require('./configs/vue3-strongly-recommended'),
+    'flat/base': {
+      plugins: { vue: module },
+      ...require('./configs/flat/base')
+    },
+    'flat/essential': {
```

Comment:
I would like to change `flat/essential`, `flat/recommended`, and `flat/strongly-recommended` to the config for Vue3. Use the `vue2-` prefix to config for Vue2 instead.